pkg/config/lighthouse/config.go

```diff
@@ -32,7 +32,7 @@ type Config struct {
 	BranchProtection branchprotection.Config `json:"branch-protection,omitempty"`
 	Orgs             map[string]org.Config   `json:"orgs,omitempty"`
 	// TODO: Move this out of the main config.
-	JenkinsOperators []JenkinsOperator `json:"jenkins_operators,omitempty"`
+	Jenkinses []JenkinsConfig `json:"jenkins_operators,omitempty"`
```

Comment:
@abayer, here is another thing I would like to adjust. I tried to get away from using the term `jenkins_operators` wherever I could. I think it is misleading. The only place left is here which unfortunately is visible to the user, since  `jenkins_operators` will be used in the config YAML. 

I initially used `jenkins`, but then noticed that other fields also did not get renamed, eg see Keeper a few lines above which still maps to `tide` in the config.  Was there the idea that the configurations should stay for some reason compatible? I find this highly confusing. We talk about Keeper, but configure "Tide" in the YAML. I think that was ok while Lighthouse was a pure Jenkins X internal thing, but if it is supposed to stand on its own feet, we need to address this somehow. 

@abayer, do you have any ideas/plans around this? Should I keep `jenkins_operators` or can I switch to `jenkins` or `jenkinses`?

Reply:
I've generally gone with the idea of keeping any field that Jenkins X cares about the same, but configuration for anything that jx doesn't care about (like Jenkins) can be different from Prow.